script/foreman-debug

```diff
@@ -303,6 +354,42 @@ add_files /etc/sysconfig/pgsql
 add_files "/var/lib/pgsql/data/pg_log/*"
 add_cmd "foreman-rake plugin:list" "plugin_list"
 
+if [ $DB_DUMP -eq 1 ]; then
+  if [ "$DB_ADAPTER" = "postgresql" ]; then
+    printv "Making full PostgreSQL database dump"
+    su postgres -c "pg_dump -Fc $DB_NAME" > "$DIR/database_pg.dump"
```

Comment:
Large deployments of Foreman (e.g. Katello and other plugins) bootup times are very slow, foreman-debug is already quite slow today and I don't want to boot the whole stack just because of this. Full dump is indeed explicit and this could be fine, but the adapter/credentials detection is still needed for the SQLs below.